src/components/Menu.js

```diff
@@ -11,9 +11,13 @@ class Menu extends Component {
             <Navbar bg="light" variant="light">
                 <Navbar.Brand href="#home"><Image src="/logo.png" height={"35"} /></Navbar.Brand>
                 <Nav className="mr-auto">
-                    <Nav.Link href="#home">Proyectos</Nav.Link>
-                    <Nav.Link href="#clientes">Clientes</Nav.Link>
+                    <Nav.Link href="#home">Clientes</Nav.Link>
+                    <Nav.Link href="#clientes">Usuarios</Nav.Link>
+                    <Nav.Link href="#reportes">Proyectos</Nav.Link>
```

Comment:
I don't think all these would be #reportes

Reply:
True